packages/x-charts/src/ScatterChart/seriesConfig/seriesProcessor.ts

```diff
@@ -36,6 +36,10 @@ const seriesProcessor: SeriesProcessor<'scatter'> = ({ series, seriesOrder }, da
           labelMarkType: 'circle' as const,
           markerSize: 4,
           ...seriesData,
+          preview: {
```

Comment:
Not sure where to document this feature. It's scatter-only, so it doesn't really make sense to document it in the zoom and pan section, but the scatter chart doesn't contain any information about zoom and preview 🤔 

Any suggestions?

Reply:
Zoom&pan page I guess. Doesn't make sense to add it to scatter page I think